_layouts/publication.html

```diff
@@ -0,0 +1,74 @@
+<!doctype html>
+<html lang="{{ site.lang }}">
+<head>
+  {% include header.html %}
+</head>
+
+<body>
+  {% include navigation.html %}
+  <main>
+    <div class="section publication-cover" style="background-color: {{ page.background_color }}">
+      {% assign download_path = "/assets/studies/" | append: page.slug %}
+      <div class="container">
+        <div class="titles">
+          <div class="publication-type">
+            {{ page.type | default: site.data.lang.text.publication.publication }}
+          </div>
+          <h1>{{ page.title_br | default: page.title }}</h1>
```

Comment:
I'd prefer a more descriptive name for the attribute, perhaps `page.title_formatted` or `page.title_html`?

Reply:
Yep, `title_formatted` sgtm.